audit-opentelemetry/chart/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+appVersion: v0.114.0
+name: opentelemetry-audit-operator
+version: 0.6.1
```

Comment:
same as in the PluginDefinition: Start with a new versioning, instead of following the original OTel plugin versioning